WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -134,15 +139,9 @@
 /* Order refunded shipping label title. The string variable shows the shipping label service name (e.g. USPS). */
 "%@ label refund requested" = "%@ label refund requested";
 
-/* Review title. Reads as {Review author} left a review on {Product}. */
-"%@ left a review on %@" = "%1$@ left a review on %2$@";
-
 /* Label for a refund on an order, which reads \"<date> via <refund method type>\", e.g. \"25 Apr 2022 via WooCommerce In-Person Payments\". Shown in a cell with a title \"Refunded\" for context */
 "%@ via %@" = "%1$@ via %2$@";
 
-/* No comment provided by engineer. */
-"%d Categories excl. %d Products" = "%1$d Categories excl. %2$d Products";
```

Comment:
Comes from the unit tests, so shouldn't be sent for translation. 🔥 

https://github.com/woocommerce/woocommerce-ios/blob/0073cca68e5a6bc351b51227780d1ce415ae8bac/WooCommerce/WooCommerceTests/Model/CouponWooTests.swift#L101